Symfony/CS/Fixer/Contrib/Php4ConstructorFixer.php

```diff
@@ -0,0 +1,167 @@
+<?php
+
+/*
+ * This file is part of the Symfony CS utility.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Symfony\CS\Fixer\Contrib;
+
+use Symfony\CS\AbstractFixer;
+use Symfony\CS\Tokenizer\Token;
+use Symfony\CS\Tokenizer\Tokens;
+
+/**
+ * @author Matteo Beccati <matteo@beccati.com>
+ */
+class Php4ConstructorFixer extends AbstractFixer
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function fix(\SplFileInfo $file, $content)
+    {
+        $tokens = Tokens::fromCode($content);
+
+        // If the code uses namespaces, then it's certainly not PHP4 ;)
+        if (count($tokens->findGivenKind(T_NAMESPACE))) {
+            return $content;
+        }
+
+        foreach ($tokens->findGivenKind(T_CLASS) as $index => $token) {
+            $classIndex = $tokens->getNonWhitespaceSibling($index, 1);
```

Comment:
Please use shorthand : `getNext...`

Will break on:
```php
class /* something */ Foo {}
```
please add it to tests and then fix